public/v4/apps/botpress.yml

```diff
@@ -0,0 +1,140 @@
+            EXTERNAL_URL: http://$$cap_appname.$$cap_root_domain
+            BP_MODULE_NLU_DUCKLINGURL: http://srv-captain--$$cap_appname-lang:8000
+            BP_MODULE_NLU_LANGUAGESOURCES: '[{"endpoint": "http://srv-captain--$$cap_appname-lang:3100"}]'
+            BP_PRODUCTION: $$cap_BOTPRESS_PRODUCTION
+            BPFS_STORAGE: $$cap_BOTPRESS_STORAGE
+            VERBOSITY_LEVEL: $$cap_BOTPRESS_VERBOSITY_LEVEL
+            BP_DECISION_MIN_CONFIDENCE: $$cap_BOTPRESS_DECISION_MIN_CONFIDENCE
+            FAST_TEXT_VERBOSITY: $$cap_BOTPRESS_FAST_TEXT_VERBOSITY
+            FAST_TEXT_CLEANUP_MS: $$cap_BOTPRESS_FAST_TEXT_CLEANUP_MS
+        depends_on:
+            - $$cap_appname-lang
+            - $$cap_appname-db
+            - $$cap_appname-cache
+        volumes:
+            - $$cap_appname:/botpress/data
+
+    $$cap_appname-lang:
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+            dockerfileLines:
+                - FROM botpress/server:$$cap_BOTPRESS_VERSION
+                - CMD bash -c "./duckling -p 8000 & ./bp lang --langDir /botpress/lang --port 3100"
+        volumes:
+            - $$cap_appname-lang:/botpress/lang
+
+    $$cap_appname-db:
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+        image: postgres:$$cap_POSTGRES_VERSION
+        environment:
+            POSTGRES_DB: $$cap_POSTGRES_DB
+            POSTGRES_PASSWORD: $$cap_POSTGRES_PASSWORD
+            POSTGRES_USER: $$cap_POSTGRES_USER
+        volumes:
+            - $$cap_appname-db:/var/lib/postgresql/data
+
+    $$cap_appname-cache:
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+            dockerfileLines:
+                - FROM redis:$$cap_REDIS_VERSION
+                - CMD redis-server --requirepass $$cap_REDIS_PASSWORD
+        volumes:
+            - $$cap_appname-cache:/data
+
+caproverOneClickApp:
+    displayName: Botpress
+    isOfficial: true
+    description: Open Source Platform For Developers To Build High-Quality Digital Assistants
+    documentation: https://botpress.com/docs
+    instructions:
+        start: |-
+            Botpress is a tool to simplify building chat bots for developers.
+            The platform puts together the boilerplate code and infrastructure you need to get a chatbot up and running and gives a complete dev-friendly platform that ships with all the tools you need to build, deploy, and manage production-grade chat bots in record time.
+        end: |-
+            Botpress has been successfully deployed! It might take few moments before it's fully started.
+
+            Before you can start using it, you need to perform some steps:
+
+            1. Go to the **HTTP Settings** of `$$cap_appname` and **Enable WebSocket Support**
+            2. Fix language files permissions by running the following command on your server `docker exec -u root -it $(docker ps --filter name=srv-captain--$$cap_appname-lang -q) chown -R botpress:botpress /botpress/lang`
```

Comment:
Wouldn't it be better to add this to CMD?

Reply:
Fixed by using `RUN` as `root` user is needed for this operation.